src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1039,6 +1051,39 @@ private enum AssignmentKind
             ForEachIterationVariable
         }
 
+        /// <summary>
+        /// Can we assign this state into this type without a warning?
```

Comment:
It feels like this doc comment is asking the opposite question of what the method is asking (should I assign *without* a warning, versus *should* I report)

Reply:
➡️ fixed comment